src/components/Content/Home/NewsBlock/NewsCard/NewsCard.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import {Button, ConfigProvider} from 'antd';
+
+import styled from "styled-components";
+
+type NewsBlockProps = {
+    image: any
```

Comment:
I think it's better to set 
`image: string `
because it's usually a path like '../src/assets/img1.jpg' that we send as `src` in `img` tag